test/test_cost_functions.jl

```diff
@@ -65,6 +65,12 @@ end
     @test first(TimeSeries.values(cost_forecast)).cost == first(data_pwl[initial_time])
 end
 
+@testset "Test MarketBidCost with single `start_up::Number` value" begin
+    expected = (hot = 1.0, warm = 0.0, cold = 0.0)  # should only be used for the `hot` value.
+    cost = MarketBidCost(; start_up = 1, no_load = rand(), shut_down = rand())
+    @test cost.start_up == expected
```

Comment:
Can you use a getter function here please? We always encourage getter functions and people usually check the tests for example code.

Reply:
I assume you mean this:

```suggestion
    @test get_start_up(cost) == expected
```

Reply:
yes